src/components/Molecules/Card/Card.style.js

```diff
@@ -74,22 +74,6 @@ const Copy = styled.div`
       font-size: 2rem;
     }
   }
-
-  /* Setting colors based on row (transparent background) vs column view */
-  @media ${({ theme }) => theme.allBreakpoints('S')} {
-    color: ${props => ((props.smallBreakpointRowLayout === true)
-    ? `${props.rowTextColor}`
-    : `${props.columnTextColor}`
-  )};
-  @media ${({ theme }) => theme.allBreakpoints('M')} {
-    color: ${props => ((props.mediumBreakpointRowLayout === true)
-    ? `${props.rowTextColor}`
-    : `${props.columnTextColor}`
-  )};
-  }
-  @media ${({ theme }) => theme.allBreakpoints('L')} {
-    color: ${props => props.columnTextColor};
-  }
```

Comment:
Can you explain the context as to why you need to remove this now?

Reply:
Ah, because the props aren't being used now? Sorry, shoulda read it all before questioning